docs/examples/use_cases/webdataset-externalsource.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>Why save and revert the global random state instead of using local random generator? https://docs.python.org/3/library/random.html#random.Random</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>